src/example-types/metricGroups/pivot.js

```diff
@@ -42,26 +85,39 @@ let buildQuery = async (node, schema, getStats) => {
 
   let statsAggs = { aggs: aggsForValues(node.values, schema) }
   // buildGroupQuery applied to a list of groups
-  let buildNestedGroupQuery = (statsAggs, groups, groupingType) =>
-    _.reduce(
+  let buildNestedGroupQuery = async (statsAggs, groups, groupingType, sort) => {
+    // Generate filters from sort column values
+    let sortAgg = await getSortAgg({ node, sort, schema, getStats })
+
+    return _.reduce(
       async (children, group) => {
         // Subtotals calculates metrics at each group level, not needed if flattening or in chart
         // Support for per group stats could also be added here - merge on another stats agg blob to children based on group.stats/statsField or group.values
         if (node.subtotals) children = _.merge(await children, statsAggs)
+        if (sortAgg) {
```

Comment:
We should add a comment explaining the idea behind the this strategy, similar to what we did above subtotals